Python/Product/PythonTools/PythonTools/Repl/Completion/ReplRemoteCompletionBroker.cs

```diff
@@ -146,11 +148,15 @@ CancellationToken token
             // this code here is just good enough to test simple cases
             // in old codebase, we used to get the expression from the analyzer
             var expression = triggerPoint.Snapshot.GetText();
-            if (expression.EndsWith(".")) {
-                expression = expression.Substring(0, expression.Length - 1);
+
```

Comment:
remove comment and probably rename `expression` for `code` to avoid potential confusion, ie it may not be an expression.

Reply:
good point